conf/machine/imx8mnevk.conf

```diff
@@ -0,0 +1,50 @@
+#@TYPE: Machine
+#@NAME: NXP i.MX8M Nano DDR4 Evaluation Kit
+#@SOC: i.MX8MM
+#@DESCRIPTION: Machine configuration for NXP i.MX8M Nano DDR4 EVK
+#@MAINTAINER:
+
+MACHINEOVERRIDES =. "mx8:mx8m:mx8mn:"
+
+require conf/machine/include/imx-base.inc
+require conf/machine/include/tune-cortexa53.inc
+
+MACHINE_FEATURES += " wifi bluetooth optee qca9377"
+
+KERNEL_DEVICETREE = "freescale/fsl-imx8mn-ddr4-evk-ak5558.dtb \
+                     freescale/fsl-imx8mn-ddr4-evk-inmate.dtb \
+                     freescale/fsl-imx8mn-ddr4-evk.dtb freescale/fsl-imx8mn-ddr4-evk-rm67191.dtb \
+                     freescale/fsl-imx8mn-ddr4-evk-rpmsg.dtb freescale/fsl-imx8mn-ddr4-evk-root.dtb"
+
```

Comment:
Maybe you can place these sorted one per line as a pretty format, your choice.

Reply:
I kind of prefer it split in one by line so it is easier to review when something is added/removed.